src/datagrid/DataGrid.stories.tsx

```diff
@@ -367,7 +367,7 @@ storiesOf("DataGrid", module)
     ))
     .add("Empty data grid", () => (
         <div style={{width: "80%"}}>
-            <DataGrid columns={normalColumns} footer={defaultFooter} />
+            <DataGrid columns={normalColumns} footer={defaultFooter} style={{height: "70vh"}} />
```

Comment:
Why is this change needed ? Datagrid is used in many places, check the impact.

Reply:
This is just for the empty DataGrid story to emphasize the center. This will not affect DataGrid in any manner.